### DISCUSSION TITLE
Options for sharing announcements/RFCs/etc.

### DISCUSSION THREAD

**Discussion by calebcartwright** [badges/shields#7017] [Ideas] [answered]
On more than one occasion we've found ourselves trying to figure out how we could best solicit feedback, communicate information, etc. with our users during certain events. We've leveraged a mix of options in the past, including Twitter, Discord, and pinned issues on GitHub, however, I'll posit that it's still an area with opportunity for improvements and suspect some/most will agree.

Part of the challenge is that we have no way to proactively _send/push_ information to our users, and at the same time there's not a great way for our users to _subscribe/pull_ information from us (at least not in an easy, ubiquitous manner).

I've been pondering two options and would be curious to hear what others think:

- Create a new Announcements (or RFC) repo under the badges org
  - Using a separate repo will make it more feasible for users to leverage GitHub's native subscribe/notify features without having to plugin to the full set of shields notifications
  - If we extended that to be more of an RFC style repo then we could also use it to request feedback/allow users the chance to weigh in
- Create a dedicated issue/discussion in badges/shields that's locked to collaborators, so that we can limit discussion to maintainers posting a message with a link to an issue or PR that has more details/dialog  
  - This would similarly help with the notification challenge that exists today, as users could subscribe to just the one issue/discussion instead of the entire repository
  - proposing such a hypothetical issue/discussion be locked to avoid cross noise and spam since there'd (hopefully) be a lot of our users subscribed to it.

cc @badges/shields-maintainers

**Comment by chris48s** [comment]
I feel like for our size of project/community the announcements thread/issue is about the right scale. A full RFC process is a bit heavy-weight. I have seen this sort of thing done elsewhere too. I don't have an example from a super high-profile project but here's a link to one that I know of: https://github.com/qcif/data-curator/issues/15

Creating an issue, locking it and pinning it also feels like a pretty low bar to get to something better than what we have now. Formalising an RFC process feels like a bigger ask.

**Reply by calebcartwright** [reply]
Yeah that makes sense to me :+1:

**Comment by Andre601** [comment]
I would've recommended creating a discussion category for news and updates (GitHub supports a news-type category where only people with write access can post new discussions), but due to GitHub's notification system would this not work that well, because people would need to opt-in to a system where they get notified about *any* new discussion created here.

That's really something GitHub has to improve, which is why I now suggested it to them: https://github.com/github-community/community/discussions/20316

**Comment by Daniel15** [comment]
> Create a new Announcements (or RFC) repo under the badges org 

For what it's worth, Microsoft uses a separate repo for ASP.NET announcements (https://github.com/aspnet/announcements) and it *seems* to work well. That project is much much larger than this one though.

**Comment by PyvesB** [comment, marked as answer]
We've since then added a blog on the website: https://shields.io/blog

**Reply by jNullj** [reply]
We could generate a static RSS feed for blog x latest items. This sounds like a classic use case for RSS.

RSS/atom has wide support and is very mature, I'm sure we could find some automated tool to generate the feed from existing blog items at build. And as it's static it won't put large pressure over infrastructure other then some bandwidth.

This has some overhead, and may not be required at all, as some RSS readers/middle-where support generating RSS feed for sites not serving one.

I'm not suggesting we rush adding such a feature, but just putting this as an option in case anyone picks that up or wants to see if this is a good direction.

(other open source projects i follow has RSS feeds, also Github has a release RSS feed as well)

**Reply by chris48s** [reply]
There already is one: https://shields.io/blog/rss.xml

and there is a `<link rel="alternate" type="application/rss+xml" href="/blog/rss.xml" title="Shields.io RSS Feed">` in the blog header HTML

**Reply by jNullj** [reply]
That's cool. We should probably put a little badge to hint at users they have that feed.
maybe at the readme, blog & homepage?
